cli/src/utils/__tests__/authWizard.test.ts

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
Let's exclude the changes of the Zai wizard improvements from this PR to isolate the changes.